### PR TITLE
Adding Side Bets

### PR DESCRIPTION
**Pull request by WhosBOakland**
Added a SideBet class, which has boolean instance variables to show if the side bet is active or if the active/inactive player won.  After the point is set, the user is asked if they want a side bet with the askForSideBet() method.  If the user says yes, the side bet is checked after each roll with the checkSideBet() method.  If the inactive or active player wins the side bet, it is displayed to the user and the balances are updated.  If the point or a 7 wasn't rolled, the user is asked if they want a side bet with the askForSideBet() method.

Additionally, I updated the wording for when a player won, so that it was clear if it was a side bet or point win.  Also cleaned up some blank println()'s to make sure the formatting was correct with the new side bet dialog.

### CODE REVIEW

File: src/main/java/com/benoakland/streetcrapscli/SideBet.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+    public boolean didInactivePlayerWin() {
+        return inactivePlayerWin;
+    }
+
+    // Methods
+    public void checkSideBet(int point, int rollResult) {
+        switch (point) {
+            case 4: case 10:
+                switch (rollResult) {
```
Comment (dotswift) [inline src/main/java/com/benoakland/streetcrapscli/SideBet.java:41]: In the areas where you switch inside of the switch, maybe you should call a private function to accomplish two things.

1. Without knowledge of craps this looks like a lot of "magic numbers" .. so the function names like handleRollResultForAScenario will help explain what that scenario is.


2. The amount of control flow here starts to become a little bit hard to follow without a large monitor. So, encapsulation would make it more readable IMO

**Review by dotswift — COMMENTED** [review]
Nice little side feature, might recommend a little cleanup but solid and quick work my friend.

File: src/main/java/com/benoakland/streetcrapscli/SideBet.java (hunk 1 above)
Comment (WhosBOakland) [reply src/main/java/com/benoakland/streetcrapscli/SideBet.java]: Encapsulated the logic for each of the switch cases in private methods for better readability.  Pushed in commit [992675b](https://github.com/WhosBOakland/street-craps-cli/pull/2/commits/992675b5ca66b25d3aeadfac6bb1843223cc3962).

**Review by dotswift — APPROVED** [review]
